### ISSUE TITLE
Database errors testing patch on Drupal 10.0.1

### ISSUE THREAD

**Issue by tyler36** [shaal/DrupalPod#104]
## Describe the bug
I was testing some Drupal 10 readiness patches today and was sometimes was getting the following error.

```php
Notice: running 'php vendor/bin/phpcs --config-set installed_paths vendor/drupal/coder/coder_sniffer' in ddev
Using config file: /var/www/html/vendor/squizlabs/php_codesniffer/CodeSniffer.conf

Config value "installed_paths" updated successfully; old value was "../../drupal/coder/coder_sniffer,../../sirbrillig/phpcs-variable-analysis,../../slevomat/coding-standard"

                                                                                                                                                                             
  Command pm:install was not found. Drush was unable to query the database. As a result, many commands are unavailable. Re-run your command with --debug to see relevant lo  
  g messages.                                                                                                                                                                
                                                                                                                                                                             

Failed to run drush en -y rename_admin_paths: exit status 1
Creating database snapshot drupalpod_20221114085254 
Created database snapshot drupalpod_20221114085254 
Your database state was locally saved, you can revert to it by typing:
ddev snapshot restore --latest
drupalpod-setup.sh script ran for 87 seconds
```

When the browser pops up, it showed the Drupal setup wizard.
I tried different modules , but it seems to happen


## To Reproduce

1. Visit https://www.drupal.org/project/rename_admin_paths/issues/3297951
2. Click DrupalPod extension and adjust following settings
 
- Drupal Core = 10.0x
- Patch = 2022-0816.... (top non-empty)
- Install profile = Standard

![d10](https://user-images.githubusercontent.com/7234392/201623153-a38964f8-4f34-432e-ad69-57d0f63b7413.jpg)

The error did _not_ display when Drupal Core `10.1.x`


## Expected behavior

- Database intialized and without errors
- "Standard" profile configured for Drupal 10.0
- Smart browser open to "Welcome page"

<!--
-->


## Screenshots

<!--
-->

### Version and configuration information (please complete the following information)

- [**Issue page**](https://www.drupal.org/project/rename_admin_paths/issues/3297951) <!-- ie. https://www.drupal.org/project/feeds/issues/3206365 -->
- **Issue branch** <!-- ie. feeds-3206365 -->
- **Drupal core version** <!-- ie. 9.3.x / 9.2.0 / etc -->
- **Project type**
- [ ] Core
- [X] Module
- [ ] Theme

## Additional context

<!--
Add any other context about the problem here. Thanks!
-->

**Comment by tyler36** [comment]
Confirmed issue on another module: https://www.drupal.org/project/rename_admin_paths/issues/3297951

- setting Drupal Core to `10.0.x` results in database error
- setting Drupal Core to `10.1.x` works (no error, Drupal installed with standard profile)

**Comment by tyler36** [comment]
Confirmed on 2 other modules:

- https://www.drupal.org/project/geolocation/issues/3308689
- https://www.drupal.org/project/superfish/issues/3299991

**Comment by tyler36** [comment]
Tested potential fix with #102 

@shaal  was able to replicate the issue with Token + D10

From Discord thread ....

> The ready-made environment (specifically of D10 + Standard) failed in setup for some reason.
When it tries to restore ddev snapshot, it gets a website that is not installed yet
